Change app endpoints to use rebrand endpoints

heytherewill opened the pull request:
## What's this?
Simple file change

## Why do we want this?
R E B R A N D 

## :squid: Permissions
Only myself.

simonrozsival commented:
Will we also be changing the URLs of other links to the app (e.g., the base URL for handoff, links to https://toggl.com/legal/privacy/, https://toggl.com/legal/terms/, and https://support.toggl.com/toggl-timer-for-ios/?

heytherewill commented:
I changed the base because we wanna get this out to users in 2.17 so that we can start monitoring if the new endpoints work fine before we ship the rest of the rebrand. I also removed the code for checking the certificates, since [it is are no longer needed](https://github.com/toggl/operations/issues/1591)
